### DISCUSSION TITLE
New Inky display size

### DISCUSSION THREAD

**Discussion by NNygren** [jayofelony/pwnagotchi#95] [Ideas]
The screen size was off on the newer InkyPHAT display that I have, so I used these parameters in the /usr/local/lib/python3.11/dist-packages/pwnagotchi/ui/hw/inky.py file and figured I'd share for those with the same display:

EDIT: actually, just copy the block from the wavesharev3.py file.

        self._layout['width'] = 250
        self._layout['height'] = 122
        self._layout['face'] = (0, 37)
        self._layout['name'] = (5, 18)
        self._layout['channel'] = (0, 0)
        self._layout['aps'] = (30, 0)
        self._layout['uptime'] = (180, 0)
        self._layout['line1'] = [0, 12, 250, 12]
        self._layout['line2'] = [0, 107, 250, 107]
        self._layout['friend_face'] = (0, 76)
        self._layout['friend_name'] = (40, 78)
        self._layout['shakes'] = (0, 108)
        self._layout['mode'] = (220, 108)
        self._layout['status'] = {
            'pos': (130, 18),
            'font': fonts.status_font(fonts.Medium),
            'max': 20
        }

**Comment by nuclearrage** [comment]
This helped me locate where the resolution config files were , now just to fix it. Thanks for posting!
